services/FormsService.ts

```diff
@@ -27,6 +27,19 @@ export class FormsService extends ApiService {
     );
   }
 
+  async downloadTraineeLtftPdf(id: String) {
```

Comment:
Hi @DorisWongNHS , I think we can improve this function.
1. Best to use `const` not `let` for the `requestConfig` object as it's not being reassigned.
2. If need `string` type, best practice to use `string` primitive - although I don't think string or any other type is needed because there is no request body for a `get` request.